appservice/src/deploy/waitForDeploymentToComplete.ts

```diff
@@ -13,12 +13,13 @@ import { localize } from '../localize';
 import { SiteClient } from '../SiteClient';
 import { delay } from '../utils/delay';
 import { nonNullProp } from '../utils/nonNull';
+import { IDeployContext } from './IDeployContext';
 
 // We get a lot of errors reported for deploy (e.g. InternalServerError), but it's possible that's just from listing the logs (which has a lot of calls to kudu), not actually a deploy failure
 // Thus we will retry a few times with exponential backoff. Each "set" of retries will take a max of about 15 seconds
 const retryOptions: retry.Options = { retries: 4, minTimeout: 1000 };
 
-export async function waitForDeploymentToComplete(context: IActionContext, client: SiteClient, expectedId?: string, token?: CancellationToken, pollingInterval: number = 5000): Promise<string> {
+export async function waitForDeploymentToComplete(context: IDeployContext, client: SiteClient, expectedId?: string, token?: CancellationToken, pollingInterval: number = 5000): Promise<void> {
```

Comment:
Any reason to change the return type? I assume nothing was using it?

Reply:
I got rid of the code using the result so there was no point to return anything anymore